package.json

```diff
@@ -28,70 +28,70 @@
   },
   "dependencies": {
     "babel-polyfill": "6.26.0",
-    "document-register-element": "1.5.0",
+    "document-register-element": "1.7.0",
```

Comment:
im hesitant in updating this without thorough testing

Reply:
cc @dvoytenko

Reply:
Noted. What does `document-register-element` do, and what kind of testing must we do? Also, is there any reason these four items must live in `dependencies` and not `devDependencies`?

Reply:
document-register-element is a polyfill for browsers that don't support custom elements, this is going to be mostly old iOS i believe.

yes, these items are in dependencies and not devDependencies since these are node modules that are compiled **with** the production runtime (meaning they are included in the code that we serve) and not a tool/development dependency

We just need to understand what changed between 1.5 -> 1.6 -> 1.7 and test on browsers that don't support custom element and ones that do

Reply:
Honestly, it might be a clean update but since it affects production code, id rather be cautious

Reply:
Gotcha, re: dependencies vs devDependencies.

Here's the commit log for `document-register-element`. About a dozen code changes between versions 1.5.0 and 1.7.0. @dvoytenko @erwinmombay, think you can look at them and see if you're worried about anything?

https://github.com/WebReflection/document-register-element/commits/master 

FWIW, out `integration_tests` shard on Travis now runs tests on 12 browsers, including old versions of iOS. (11.0, 10.0, and 9.0). See https://travis-ci.org/ampproject/amphtml/jobs/318328159

Reply:
@erwinmombay I've reverted the change to the `document-register-element` version. I agree that it should go in as a separate PR that's easy to revert if needed.

PTAL.